Switch to official geerlingguy.docker role

priteau opened the pull request:
The role was recently updated to use ansible_facts [1].

[1] https://github.com/geerlingguy/ansible-role-docker/commit/c0ddaa9d646e7dc7e10152559446c7d6ce378136

mnasiadka reviewed (commented):
Is there a reason why we use this, not ansible-collection-kolla docker role?

priteau commented on etc/kayobe/ansible/requirements.yml:
```diff
@@ -30,8 +30,7 @@ roles:
   - name: geerlingguy.docker
-    src: https://github.com/stackhpc/ansible-role-docker.git
-    version: stackhpc/7.0.1.1
+    version: 7.8.0
```
Actually the comment refers to the ansible-gitlab-runner role.

priteau commented:
> Is there a reason why we use this, not ansible-collection-kolla docker role?

It appears to be used only for the `deploy-github-runner.yml` playbook.

jackhodgkiss commented:
I think we can merge this however I would like to move to ansible-collection-kolla docker to help reduce dependencies.

priteau committed:
Switch to official geerlingguy.docker role

The role was recently updated to use ansible_facts [1].

[1] https://github.com/geerlingguy/ansible-role-docker/commit/c0ddaa9d646e7dc7e10152559446c7d6ce378136
